src/api/movie.ts

```diff
@@ -20,13 +33,16 @@ export default {
           `${url}movie?cb=&quality=720p,1080p,3d&page=1&imdb=${imdbId}`,
         ),
       getYtsDownload: (imdbID: string) =>
-        axios.get(`${yts}list_movies.json?query_term=${imdbID}&limit=1`),
-      getByGenre: (genre: [], page: number, sort = 'seeds') =>
         axios.get(
-          `${url}list?sort=${sort}&short=1&cb=&quality=720p,1080p,3d&page=${page}&genre=${genre}`,
+          `${TORRENT_API_URL}list_movies.json?query_term=${imdbID}&limit=1`,
         ),
+      getByGenre: async (genre: [], page: number, sort = 'seeds') => {
```

Comment:
```suggestion
      getByGenre: async (genre: string[], page: number, sort = 'seeds') => {
```

It's always better to be as specific as possible with the types.

Reply:
Nice catch!